src/Microsoft.AspNetCore.ResponseCaching/Interfaces/IResponseCachingResponseCacheabilityValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.ResponseCaching
+{
+    public interface IResponseCachingResponseCacheabilityValidator
```

Comment:
I thought about that but the problem is that if you only need to override one of the cacheability checks, you still need to implement both. The argument can also be made from the other side where if you use need to use both overrides, now you need two different implementations. So we just need to pick our poison.